docs/content/index.md

```diff
@@ -19,7 +19,7 @@ Moreover, Maesh is opt-in by default, which means that your existing services ar
 Maesh does not use any sidecar container but handles routing through proxy endpoints running on each node.
```

Comment:
Generel question:

should we add a "deprecates" section or something like that?

Reply:
I think we could document deprecated flags in the `install.md`, where they used to be mentioned. 
Something like this
```
# SMI Mode (Deprecated)

Maesh supports the SMI specification which defines a set of custom resources to provide a fine-grained control over instrumentation, routing and access control of east-west communications.

To enable SMI, install maesh in SMI mode by setting the smi.enable helm chart option to true.

helm install maesh --namespace=maesh maesh/maesh --set smi.enable=true`
The smi.enable option makes Maesh process SMI resources.

!!! Note This option is deprecated. Please consider using `--acl` instead.
```